lib/matplotlib/offsetbox.py

```diff
@@ -1245,6 +1279,9 @@ def _get_anchored_bbox(self, loc, bbox, parentbbox, borderpad):
 class AnchoredText(AnchoredOffsetbox):
     """
     AnchoredOffsetbox with Text.
+
+    This is a convenience class. Technically, it is just an `AnchoredOffsetbox`
+    containing a `TextArea`.
```

Comment:
last sentence basically duplicates the first line...

Reply:
The first line gives the intent of the class. The last sentence describes how it it realized. I think both are helpful.

Reply:
Something like `Convenience subclass of `AnchoredOffsetbox` for holding a `TextArea`.` would be enough, and fit in one line?

Reply:
I don't think that catches the essence. From the outside it's really created like a text object: the text `s` is the first parameter. Only if I'm to operate on it further, it helps to know that it's ans `AnchoredOffsetbox` containing a `TextArea`.

If you strictly want a single line, I'd rather go back to the original docstring.

TBH, I don't really understand all this offsetbox business. Maybe it's just a lack of documentation, but it feels overly complicated for what it's trying to achieve.

Reply:
I don't understand it completely either, but I think I would rather leave things as is and first clarify the actual intent.

Reply:
Reverted for now. Thanks for the discussion.